api/src/main/java/org/openmrs/api/db/FullTextSessionFactoryImpl.java

```diff
@@ -7,37 +7,31 @@
  * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
  * graphic logo is a trademark of OpenMRS Inc.
  */
-package org.openmrs;
+package org.openmrs.api.db;
 
-import org.hibernate.Session;
 import org.hibernate.SessionFactory;
 import org.hibernate.search.FullTextSession;
 import org.hibernate.search.Search;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationEventPublisher;
 import org.springframework.stereotype.Component;
 
-/**
- * An instance of this class provides a factory for obtaining {@link FullTextSession} instances.
- * Having this factory as a spring bean provides a mechanism that allows external code and modules
- * to advise the factory. It is highly recommended to use this factory to create instances of the
- * {@link FullTextSession} rather than directly calling {@link Search#getFullTextSession(Session)}
- * for proper functionality.
- * 
- * @since 2.2.1
```

Comment:
Did you intentionally remove the since annotation?
Could we have a description to replace the one you removed?